### PR TITLE
Add Glaze library

### PR DESCRIPTION
**Pull request by stephenberry**
Add [Glaze](https://github.com/stephenberry/glaze) JSON library

### CODE REVIEW

File: etc/config/c++.amazon.properties [hunk 1]
```diff
@@ -3506,6 +3506,15 @@ libs.gemmlowp.versions=trunk
 libs.gemmlowp.versions.trunk.version=trunk
 libs.gemmlowp.versions.trunk.path=/opt/compiler-explorer/libs/gemmlowp/trunk
 
+libs.glaze.name=glaze
+libs.glaze.description=JSON, reflection, and interface library
+libs.glaze.versions=trunk:198
+libs.glaze.url=https://github.com/stephenberry/glaze
+libs.glaze.versions.trunk.version=trunk
+libs.glaze.versions.trunk.path=/opt/compiler-explorer/libs/glaze/trunk
```
Comment (mattgodbolt) [inline etc/config/c++.amazon.properties:3514]: There's no trunk glaze installed in your infra PR; there's just the `v1.9.8`. I'll quickly hack that in too :)

Currently...
```
$ ls /opt/compiler-explorer/libs/glaze/
v1.9.8
```

Comment (mattgodbolt) [reply etc/config/c++.amazon.properties:3514]: now installed!

File: etc/config/c++.amazon.properties [hunk 2]
```diff
@@ -3506,6 +3506,15 @@ libs.gemmlowp.versions=trunk
...
+libs.glaze.versions.198.version=1.9.8
+libs.glaze.versions.198.path=/opt/compiler-explorer/libs/glaze/1.9.8
```
Comment (mattgodbolt) [inline etc/config/c++.amazon.properties:3516]: seems this needs to be v1.9.8; will fix and merge :)
```suggestion
libs.glaze.versions.198.path=/opt/compiler-explorer/libs/glaze/v1.9.8
```

**Comment by partouf** [comment]
this is live, but it is incorrectly configured, `/include` should be added to the paths, can you send us a PR for that?

**Comment by stephenberry** [comment]
Pardon me, I'm unfamiliar with the setup. What are you referring to concerning `paths`? Is this another file or within `c++.amazon.properties`?

**Comment by partouf** [comment]
`c++.amazon.properties` yes,
example:
`libs.glaze.versions.trunk.path=/opt/compiler-explorer/libs/glaze/trunk`

**Pull request merged**